conf/config-default.yaml

```diff
@@ -115,8 +120,16 @@ apisix:
   enable_resolv_search_opt: true  # enable search option in resolv.conf
   ssl:
     enable: true
-    enable_http2: true
-    listen_port: 9443
+    # listen: 9443                # APISIX listening port in https.
```

Comment:
I think we should only support table format for the new configuration, this will make it easier to understand.

Reply:
ok，I have removed it in the `config-default.yaml`.